.eslintignore

```diff
@@ -3,7 +3,7 @@ tag-admin/dist/
 translate/dist/
 coverage/
 docs/_build/
-docs/venv/
+docs/*venv/
```

Comment:
By default, `uv` creates a hidden `.venv` folder. I thought it would be nice to keep covering also `venv`, since I got bitten by this (same in `.gitignore`).